modules/installation-aws-permissions-iam-shared-vpc.adoc

```diff
@@ -0,0 +1,30 @@
+// Module included in the following assembly:
+// * installing/installing_aws/installing-aws-account.adoc
+
+:_content-type: REFERENCE
+[id="installation-aws-permissions-iam-shared-vpc_{context}"]
+= Trust policy for installing into a shared VPC
```

Comment:
@bscott-rh Great update in this section, it's clear.
one comment is about the title, as permission policy was added into this section, so this part contains _trust policy + permission policy_, how about
```suggestion
= The role policy for installing into a shared VPC
```

@patrickdillon WDYT?

Reply:
Thanks, I have slightly reworded the title with this in mind.